Modules/CodeTrek/Http/Controllers/CodeTrekController.php

```diff
@@ -32,11 +34,18 @@ public function index(Request $request)
         $applicantData = $this->service->getCodeTrekApplicants($request->all());
         $applicants = $applicantData['applicants'];
         $statusCounts = $applicantData['statusCounts'];
+        $start_date = Carbon::parse($request->application_start_date) ?? today()->subYear();
+        $end_date = Carbon::parse($request->application_end_date) ?? today();
+        $reportApplicationCounts = CodeTrekApplicant::select(\DB::Raw('DATE(start_date) as date, COUNT(*) as count'))
+            ->whereDate('start_date', '>=', $start_date)
+            ->whereDate('start_date', '<=', $end_date)
+            ->count();
```

Comment:
Feedbacks
1. naming convention of variable is incorrect
2. When no date filter is applied then the count is coming as 0. 

```suggestion
        $startDate = Carbon::parse($request->input('application_start_date', today()->subYear()));
        $endDate = Carbon::parse($request->input('application_end_date', today()));
        $reportApplicationCounts = CodeTrekApplicant::whereDate('start_date', '>=', $startDate)
            ->whereDate('start_date', '<=', $endDate)
            ->count();
```